Support load-balance GRPC client with Kubenetes selector (#69)
apache/skywalking-satellite: docs/en/setup/examples/grpc-load-balance-client/README.md

# GRPC load balance client

> GRPC client support connect to multiple server address, and using round-robin to load-balance server before send each request.

Reviewer: ```suggestion
GRPC client support connect to multiple server address, and use `round-robin` policy for load-balance.
```
Does this base on per streaming or per message?

Author: Per streaming.

## Server Discovery

Support two ways to locate the server list:

> 1. Static server list: Define the server address list.

Reviewer: Why do we need to mention `static list again`? I remember we used to have one already.

Author: Here is an example to let users know how to define multiple servers, `static server lists` and `Kubernetes selector` are two different ways.

Reviewer: I see. Then you missed the menu update.

Author: Added.

      kubernetes_config:
        # The kubernetes API server address, If not define means using in kubernetes mode to connect
        api_server: http://localhost:8001/
        # The kind of api
        kind: endpoints
        # Support to lookup namespaces
        namespaces:
          - default
        # The kind selector
        selector:
          label: app=productpage
        # How to get the address exported port
        extra_port:
          port: 9080
      # The TLS switch
      enable_TLS: ${SATELLITE_GRPC_ENABLE_TLS:false}
      # The file path of client.pem. The config only works when opening the TLS switch.
      client_pem_path: ${SATELLITE_GRPC_CLIENT_PEM_PATH:"client.pem"}
      # The file path of client.key. The config only works when opening the TLS switch.
      client_key_path: ${SATELLITE_GRPC_CLIENT_KEY_PATH:"client.key"}
      # InsecureSkipVerify controls whether a client verifies the server's certificate chain and host name.
      insecure_skip_verify: ${SATELLITE_GRPC_INSECURE_SKIP_VERIFY:false}
      # The file path oca.pem. The config only works when opening the TLS switch.
      ca_pem_path: ${SATELLITE_grpc_CA_PEM_PATH:"ca.pem"}

>       # How frequently to check the connection

Reviewer: Add unit please.
